data/adiabatic-conditions/ascii-data/isentrope_properties.txt

```diff
@@ -0,0 +1,504 @@
+# POINTS: 501 
```

Comment:
It is "Birch-Murnaghan" - you had me laughing out loud on this one.

While I agree that "isentrope" is not a great name, the equation of state is not the only parameter either of course. I see this as one of possibly many example profiles.